README.md

```diff
@@ -90,7 +90,7 @@ See more about the project testsuite in a
 | `workspace/didChangeWorkspaceFolders` |                    |
 | `workspace/didChangeConfiguration`    | :white_check_mark: |
 | `workspace/didChangeWatchedFiles`     |                    |
-| `workspace/symbol`                    |                    |
+| `workspace/symbol`                    | :white_check_mark: |
```

Comment:
but we haven't implemented workspace symbol request yet, have we?
We provide only `document/symbol`

Reply:
good catch, fixed